src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -114,7 +113,7 @@ public Controls(Subsystems s) {
 
         // driveController.getBackButton().whenPressed(() -> driveController.activate(SECONDARY));
 
-        boolean comp = Robot.getInstance().isCompetition();
+        boolean comp = Robot.instance.isCompetition();
```

Comment:
Need to think a bit more about how this ought to be communicated, as both `Robot.getInstance()` and `Robot.instance` are 'code smells' that we should get rid of somehow. It's valuable have this PR highlight where they're used, to prompt thought about how to get rid of them for something better.

Reply:
i think robot.getinstance() makes sense because you only have one and its a way to access something
its hard to do which is good, but it should exist nonetheless